docs/user-manual/index.md

```diff
@@ -1356,6 +1369,28 @@ group in its group map.  It updates membership in the user group,
 which indirectly updates the membership in the product
 configuration.
 
+### Working with Non-Email-based Usernames
+
+On the Adobe Admin Console, you can configure a domain to use email-based user login names or non-email-based usernames.  We often user the term Username based login to mean non-email-based-login.  Username-based login can be used when email addresses are expected to change often or your organization does not allow email addresses to be used for login.
```

Comment:
This needs some work.  See if you can work in the following information:
* This only applies to Federated ID domains, that use SAML-based SSO with an IdP.
* The terms Username-based login and Email-based login come directly from the SSO configuration settings for the domain, where there is a configuration field called "User Login Setting" that can be set to "Email address" or "Username" (see snapshot below)
* Whether to use username-based login or email-based login depends on a company's overall identification strategy, and are outside the scope of this document.

![sso federation snapshot](https://cloud.githubusercontent.com/assets/24463201/24022211/4cc4e14e-0a62-11e7-9b74-4b5e7747d147.png)